db/migrate/20240813192728_create_ratings.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRatings < ActiveRecord::Migration[7.1]
+  def change
+    create_table :ratings do |t|
+      t.references :user, null: false, foreign_key: true
+      t.references :book, null: false, foreign_key: true
+      t.integer :rating
```

Comment:
add null false validation,  add index unique for book and user 

https://stackoverflow.com/questions/34424154/rails-validate-uniqueness-of-two-columns-together